CHANGELOG.unreleased.md

```diff
@@ -15,6 +15,7 @@
 
 - [Proxy/deploy] Fix `no such proxy ok` error on a failure trial start (PR [#5196](https://github.com/vatesfr/xen-orchestra/pull/5196))
 - [VM/snapshots] Fix redirection when creating a VM from a snapshot (PR [#5213](https://github.com/vatesfr/xen-orchestra/pull/5213))
+- [User] fix error when changing password [#5218](https://github.com/vatesfr/xen-orchestra/issues/5218) (PR [#5221](https://github.com/vatesfr/xen-orchestra/pull/5221))
```

Comment:
Please specify the error.

Reply:
Is there only an error or was password changing completely broken?

Reply:
There is only an error.

Reply:
But the password is correctly changed nevertheless?

Reply:
Yes.